genesyscloud/resource_genesyscloud_routing_email_route.go

```diff
@@ -124,7 +124,7 @@ func resourceRoutingEmailRoute() *schema.Resource {
 			"from_email": {
 				Description: "The sender email to use for outgoing replies.",
 				Type:        schema.TypeString,
-				Optional:    true,
+				Required:    true,
```

Comment:
The problem here is that the API does not mark the from_email as required.  It actually is required as the underlying API will complain that a value must be between 1 and 256.

Reply:
Typical.